boto/storage_uri.py

```diff
@@ -118,23 +118,6 @@ def connect(self, access_key_id=None, secret_access_key=None, **kwargs):
                 self.provider_pool[self.scheme] = self.connection
             elif self.scheme == 'gs':
                 from boto.gs.connection import GSConnection
-                # Use OrdinaryCallingFormat instead of boto-default
-                # SubdomainCallingFormat because the latter changes the hostname
-                # that's checked during cert validation for HTTPS connections,
-                # which will fail cert validation (when cert validation is
-                # enabled).
-                #
-                # The same is not true for S3's HTTPS certificates. In fact,
-                # we don't want to do this for S3 because S3 requires the
-                # subdomain to match the location of the bucket. If the proper
-                # subdomain is not used, the server will return a 301 redirect
-                # with no Location header.
-                #
-                # Note: the following import can't be moved up to the
-                # start of this file else it causes a config import failure when
-                # run from the resumable upload/download tests.
-                from boto.s3.connection import OrdinaryCallingFormat
-                connection_args['calling_format'] = OrdinaryCallingFormat()
```

Comment:
Can't say I know the reason for removing this

Reply:
If we kept this, then we'd always use the ordinary calling format regardless of the user's boto setting. I removed the comment because it appears to be false: gsutil seems to work just fine with GCS virtual hosted style endpoints.